ffi/system-command/win32ui.dart

```diff
@@ -31,6 +31,7 @@ typedef MessageBoxDart = int Function(
   int type,
 );
 
+// ignore_for_file: constant_identifier_names
```

Comment:
Maybe add this to the `analysis_options.yaml` instead?

Reply:
It looks like John added it, so I dropped the redundant comments.

Reply:
SGTM